apps/exchange/config/test.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+config :exchange, message_bus_adapter: Exchange.Adapters.TestEventBus, environment: Mix.env()
```

Comment:
why do we need to add this bit to the config ?
`environment: Mix.env()` this can always be inferred using Mix.env() on the code no ?
also if we're doing this on the config/test.exs we already know that the environment: :test

am I missing something ?

Reply:
I tried to follow the existing configs, namely the config of eventbus. 
However, you are right I will remove this assign.